<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RUserWindow</class>
 <widget class="QDialog" name="RUserWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>275</width>
    <height>164</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Регистрация</string>
  </property>
  <widget class="QPushButton" name="pb_registr">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>120</y>
     <width>131</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Зарегистрироваться</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_uname">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>20</y>
     <width>131</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_passu">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>50</y>
     <width>131</width>
     <height>22</height>
    </rect>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::EchoMode::Password</enum>
   </property>
  </widget>
  <widget class="QLabel" name="lab_uname">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Имя (ник EN)</string>
   </property>
  </widget>
  <widget class="QLabel" name="lab_passu">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>58</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>Пароль</string>
   </property>
  </widget>
  <widget class="QLabel" name="lab_passu1">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Повторите пароль</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_passu1">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>80</y>
     <width>131</width>
     <height>22</height>
    </rect>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::EchoMode::Password</enum>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
